mdpow/tests/test_automated_dihedral_analysis.py

```diff
@@ -43,40 +46,79 @@ def SM25_tmp_dir(self, molname_workflows_directory):
         return dirname
 
     @pytest.fixture(scope="function")
-    def atom_indices(self, SM25_tmp_dir):
-        atom_group_indices = dihedrals.dihedral_indices(dirname=SM25_tmp_dir, resname=self.resname)
+    def mol_sol_data(self, SM25_tmp_dir):
+        u = dihedrals.build_universe(dirname=SM25_tmp_dir)
+        mol, solute = dihedrals.rdkit_conversion(u=u, resname=resname)
+        return mol, solute
+
+    @pytest.fixture(scope="function")
```

Comment:
function scope is the default — just remove when you use the default (here and elsewhere)

Reply:
There is an issue open for reviewing tests, so I will handle this here and leave a reminder in that issue to handle elsewhere.